### PR TITLE
Tag CPUTime.jl v0.0.6

### PR DESCRIPTION
**Pull request by attobot**
Repository: [schmrlng/CPUTime.jl](https://github.com/schmrlng/CPUTime.jl)
Release: [v0.0.6](https://github.com/schmrlng/CPUTime.jl/releases/tag/v0.0.6)
Travis: [Travis Build Status](https://travis-ci.org/schmrlng/CPUTime.jl/builds/399345726)
Diff: [vs v0.0.5](https://github.com/schmrlng/CPUTime.jl/compare/7932f253e557d3203c8d427381681b7586ef37c2...b464a77f5ae1fdb299d746c833ee44f9351adce6)
`requires` vs v0.0.5: 
```diff
--- 0.0.5/requires
+++ 0.0.6/requires
@@ -1,2 +1,2 @@
 julia 0.3
-Compat
+Compat
```
cc: @schmrlng

Please make sure that:
- CI passes for supported Julia versions (if applicable).
- Version bounds reflect minimum requirements.

### CODE REVIEW

**Comment by schmrlng** [comment]
Whoops, don't merge this yet -- stand by for an attobot update.

**Comment by schmrlng** [comment]
This time around should be good to go.

File: CPUTime/versions/0.0.6/requires [hunk 1]
```diff
@@ -0,0 +1,2 @@
+julia 0.3
```
Comment (ararslan) [inline CPUTime/versions/0.0.6/requires]: METADATA registrations currently require a minimum Julia version of 0.6, which is why Travis is failing here.

**Comment by schmrlng** [comment]
Sure, I've bumped the minimum version requirement to Julia 0.6.

**Comment by ararslan** [comment]
From the Travis log:

> ERROR: LoadError: New tag 0.0.6 of package CPUTime requires julia 0.6.0, but version 0.0.1 of CPUTime requires julia 0.3.0-. Use a new minor package version when support for an old version of Julia is dropped. Re-tag the package as 0.1.0 using `PkgDev.tag("CPUTime", :minor)`.

Note though that you don't have to use PkgDev, you can still use Attobot. Just delete the 0.0.6 tag and make a 0.1.0 tag, and Attobot should open a new PR. Then I can close this one for you.

**Comment by schmrlng** [comment]
Great, thanks for your help (see #15502).

**Pull request closed without merging**